bundle/src/main/java/com/adobe/acs/commons/synth/SynthesizerServlet.java

```diff
@@ -0,0 +1,43 @@
+package com.adobe.acs.commons.synth;
+
+import org.apache.felix.scr.annotations.sling.SlingServlet;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.SlingHttpServletResponse;
+import org.apache.sling.api.request.RequestParameter;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.api.servlets.SlingSafeMethodsServlet;
+
+import javax.servlet.ServletException;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@SlingServlet(
+        paths = "/services/synthesizer"
```

Comment:
can you change to `/bin/acs-commons/synthesizer` all our other path-based servlets are registered under that common path prefix